mos-platform/rpc8e/crt0.S

```diff
@@ -0,0 +1,46 @@
+;
+; Copyright (c) 2023 Adrian "asie" Siekierka
```

Comment:
As far as is possible, we've licensed everything in the compiler and SDK under Apache 2.0 with LLVM exceptions. There's a bit of MIT floating around in the SDK, but I'd like to get rid of it as soon as we can get llvm-libc up an running; having a simple licensing structure is a definite benefit.

Would it be acceptable to license these files under those terms?

Reply:
With the exception of `librpc8e/src/display.c` (where some snippets of code are derived from RPC8/e support work by L. Adamson done for cc65 back in 2012), it is acceptable to license all files under these terms.

Reply:
I have relicensed the crt0 code to match; what should be done about the non-crt0 library?

Reply:
Actually, the cc65 license is compatible with Apache 2.0; we're free to relicense code under that license so long as we follow its terms, which are just that we need an attribution notice for the source of that code, and a notice saying that they were modified from their original versions. We've done that with quite a few files already; most of the SDK headers are based on the cc65 equivalents.

Reply:
Done. This leaves the examples, where I feel a CC0 license (or similar, such as 0BSD) would be justifiable to encourage reuse of such sample code; but I can relicense those too, of course.

Reply:
Eh, that's fine, the risk there is low. The only thing you really gain with Apache2+LLVM over that is patent use and clear contributor relicensing clauses, but I wouldn't expect the examples there to be subject to continuous development.

I've mostly just tried to keep the licensing structure of the project in line with the implicit expectations folks should have with a project like this: it shouldn't be possible to start swinging patent or copyright law around to kill the project or annoy contributers, and there shouldn't be any onerous burdens on those using the output of the compiler, including any code compiled into the SDK. The LLVM exceptions help with that a lot, since they explicitly free compiled versions of the SDK from needing to bundle the license, which would otherwise be quite onerous for retro gamedev.

In practice, it probably doesn't matter; you'd have to be pretty bored and angry to sue a bunch of retro hackers. But, it hasn't been *too* difficult to structure things well "above board"; so far so good, anyway!

Reply:
I'm glad we're on the same page with regards to choice of license! I've chosen the zlib license for my toolchain projects in the past, but I might migrate to Apache 2.0+LLVM myself